frontend/src/pages/RideDetails/RideDetails.css

```diff
@@ -88,7 +88,7 @@
 }
 
 ul li{
```

Comment:
I wasn't able to access the ride details page(s) yet, but I think it's best for now to not use predefined css classes because they easily conflict with styles elsewhere.

Reply:
I should've requested for your review for my previous change. I'll be more careful about that problem. Also, I figured out the way of doing it correctly with semantic. I'll update it in my later pull request.